src/redux/store.js

```diff
@@ -0,0 +1,12 @@
+import { configureStore } from '@reduxjs/toolkit';
+import { booksSlice } from './books/booksSlice';
+import { categoriesSlice } from './categories/categoriesSlice';
+
+const store = configureStore({
+  reducer: {
+    book: booksSlice,
+    categories: categoriesSlice,
+  },
+});
```

Comment:
- [ ] Hi @jecode93 you did a really great job by implementing the features given in the requirement. But it seems like instead of importing `reducers` from the **booklice** and `categoriesSlice`, you are importing the slice. By default, once you exported the `booksSlice.reducer` from `booksSlice` and **categories. reducer** from the `categoriesSlice` file then you can directly import *booksReducer* and *categoriesReducer* because behind the scene when you `export default books.reducer`, it translates as `export default function booksReducer() {}`. I hope you will understand it by doing research and learn more about it. Kindly fix the issue and submit your project again.

Reply:
@skyv26 Yes, yes you right... Thank you very much for you review! I appreciate.

Reply:
@jecode93 Your Welcome